### DISCUSSION TITLE
Translating into further languages

### DISCUSSION THREAD

**Discussion by manu20247** [helloSystem/hello#571] [General]
Hello, I would like to know how to contribute to the translation of the system into Brazilian Portuguese and I would like to know if the system has a channel on Telegram.  Thanks.

**Comment by probonopd** [comment]
Hello @manu20247. Thanks for your offer to help, but currently we are not looking to add further translations, as we have to figure out more basic stuff. We are still in the 0.x phase. Thanks for your understanding.

**Reply by manu20247** [reply]
Hello dear, thank you for answering me, I look forward to helping you with something, I really want to contribute to the project.    I know some designer friends, if necessary, we can create icons and mouse cursors with their own identity for the system while I am unable to help financially.   Or I can help with ideas, I don't understand much about programming and when I create some code I use AI 😂😂😂 it helps a lot.   But I won't take up your time, thank you very much.
